readme.md

```diff
@@ -115,6 +115,11 @@ You can determine what is shown based on what a user is authorised to see by
 using the `When` component.
 
 ```jsx
+import { When } from 'pundit'
+import PostPolicy from 'src/policies/post.policy.js'
+
+...
```

Comment:
what about 

```js
// ...
```

instead?

Reply:
Good idea - makes it clearer that it is not code. Updated.

Reply:
I've also added `return ()` around the JSX elements to it's a bit clearer when the JavaScript ends and the JSX elements start.